pkg/kubeinteraction/events.go

```diff
@@ -3,7 +3,7 @@ package kubeinteraction
 import (
 	"context"
 
-	"github.com/google/go-github/v45/github"
+	"github.com/google/go-github/v47/github"
```

Comment:
47 😄

Reply:
i think we get the infamous too slow to report status :(